orderbook/src/main.rs

```diff
@@ -50,6 +50,13 @@ struct Arguments {
         parse(try_from_str = shared::arguments::duration_from_seconds),
     )]
     min_order_validity_period: Duration,
+
+    /// Don't use the trace_callMany api that only some nodes support to check whether a token
+    /// should be denied.
+    /// Note that if a node does not support the api we still fallback to the less accurate call
+    /// api.
+    #[structopt(long, env = "SKIP_TRACE_API")]
+    skip_trace_api: bool,
```

Comment:
I'm using the negation "skip" here instead of "enable" because you cannot define a default value for boolean arguments but I want the default to be that the api is enabled.